pyrolite/plot/templates/QAP.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from ...util.plot.axes import init_axes
+from ...util.classification import QAP as QAP
+from ...util.meta import sphinx_doi_link, update_docstring_references, subkwargs
+from ...util.log import Handle
+
+logger = Handle(__name__)
+
+
+@update_docstring_references
+def QAP(ax=None, add_labels=False, color="k", **kwargs):
+    """
+    IUGS QAP ternary classification
+    [#ref_1]_ [#ref_2]_.
+
+    Parameters
+    -----------
+    name : :class:`str`
+        A name for the classifier model.
+    axes : :class:`list` | :class:`tuple`
+        Names of the axes corresponding to the polygon coordinates.
+    fields : :class:`dict`
+        Dictionary describing indiviudal polygons, with identifiers as keys and
+        dictionaries containing 'name' and 'fields' items.
+
+    References
+    -----------
+}    .. [#ref_1] Streckeisen, A. Classification and nomenclature of plutonic rocks
```

Comment:
There is a leading `}` which snuck in here and is breaking a few things. Most should be fine when this is removed.